src/audio/component.c

```diff
@@ -62,7 +62,7 @@ int comp_set_state(struct comp_dev *dev, int cmd)
 	int requested_state = comp_get_requested_state(cmd);
 
 	if (dev->state == requested_state) {
-		comp_info(dev, "comp_set_state(), state already set to %u",
+		comp_warn(dev, "comp_set_state(), state already set to %u",
```

Comment:
This is a bit unexpected and does not match the git commit message. I can't comment whether this should be a warning or not and we'd need to get ack from author of this component.c code.

Reply:
ok, let me change to info, from performance view, there is no difference for info and warn.